packages/bitcore-wallet-client/src/lib/common/constants.ts

```diff
@@ -21,23 +21,65 @@ export const Constants = {
     REQUEST_KEY_AUTH: 'm/2' // relative to BASE
   },
   BIP45_SHARED_INDEX: 0x80000000 - 1,
+
```

Comment:
I think it's safe to remove the COINS array https://github.com/bitpay/bitcore/pull/3400

Reply:
When your PR is merged in I will pull in the latest and make revisions to this PR as needed.

Reply:
I confirmed that the COINS array can be removed and I removed it.